meinberlin/static/embed.js

```diff
@@ -138,6 +138,7 @@ $(document).ready(function () {
     $error.text(text)
     $error.append($close)
     $error.prependTo($('#embed-status'))
+    $close.attr('title', django.gettext('Close'))
```

Comment:
Access to title attribute information is not supported uniformly by screen readers. See: https://www.w3.org/TR/html/dom.html#the-title-attribute